scripts/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/elastic/ecs/scripts
```

Comment:
Great to see versioning on these. 

++ on moving the template.go und cmd above.